ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue

```diff
@@ -1,10 +1,10 @@
 <template>
   <v-card flat rounded id="submitting-party" class="mt-8 pa-8 pr-6">
     <v-row no-gutters>
-      <v-col cols="12" sm="2">
+      <v-col :cols="isMhrTransfer ? '3' : '12'"  :sm="isMhrTransfer ? '0' : '2'">
         <label class="generic-label" :class="{'error-text': validate}">Add Submitting Party</label>
       </v-col>
-      <v-col cols="12" sm="10" class="px-1">
+      <v-col :cols="isMhrTransfer ? '9' : '12'"  :sm="isMhrTransfer ? '0' : '10'" class="px-1">
```

Comment:
Darn, this is a bit annoying..

Reply:
fiddly design bits but not too bad

Reply:
Is there a way to keep it the same? I saw the comment from the designers, i think it should be consolidated at the design level.

Reply:
We cannot be this granular, it might break styling elsewhere....

Reply:
@owillborn maybe you could check with designers about this one?

Reply:
Maybe show both pages with different columns and see which one it should be.

Reply:
@owillborn thanks, what was the outcome? I see there are no changes in this file.

Reply:
Perfect